synapse/storage/prepare_database.py

```diff
@@ -27,7 +27,7 @@
 
 # Remember to update this number every time a change is made to database
 # schema files, so the users will be informed on server restarts.
-SCHEMA_VERSION = 55
+SCHEMA_VERSION = 56
```

Comment:
what's the thinking behind the schema bump?

Reply:
Mainly that this stops people from rolling back, which will cause `membership` column to not be correctly updated.